articles/python/azure-sdk-sovereign-domain.md

```diff
@@ -67,11 +71,13 @@ stack_cloud = get_cloud_from_metadata_endpoint("https://contoso-azurestack-arm-e
 # https:// is optional in the URL but required on the endpoint.
 credential = DefaultAzureCredential(authority=stack_cloud.endpoints.active_directory)
 
-resource_client = ResourceManagementClient(credential, subscription_id,
+resource_client = ResourceManagementClient(
+    credential, subscription_id,
     base_url=stack_cloud.endpoints.resource_manager,
-    credential_scopes=[cloud.endpoints.resource_manager + ".default'"])
```

Comment:
`resource_manager` may differ from `active_directory_resource_id` and is not valid `resource` in Azure Stack.